core/designsystem/src/main/java/com/yveskalume/eventcademy/core/designsystem/components/LinkText.kt

```diff
@@ -146,7 +145,7 @@ object LinkedTextDefaults {
         backgroundColor: Color = Color.Unspecified,
         disabledBackgroundColor: Color = backgroundColor,
         pressedBackgroundColor: Color = textColor.copy(
-            alpha = LocalRippleTheme.current.rippleAlpha().pressedAlpha
```

Comment:
Why to you change this ?

Reply:
I tried to run the App and i've got this error : 

e: file:///home/geek-pastor/AndroidStudioProjects/EventCademy/core/designsystem/src/main/java/com/yveskalume/eventcademy/core/designsystem/components/LinkText.kt:149:21 '@property:Deprecated(...) val LocalRippleTheme: ProvidableCompositionLocal<RippleTheme>' is deprecated. RippleTheme and LocalRippleTheme have been deprecated - they are not compatible with the new ripple implementation using the new Indication APIs that provide notable performance improvements. For a migration guide and background information, please visit developer.android.com.